app/controllers/api/v2/people_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    class PeopleController < ApiController
+      before_action :validate_include_params, only: :index
```

Comment:
```suggestion
      before_action :validate_include_params
```

Reply:
Seems to me that you'll be removing this anyway :)

Reply:
I think it's good to have 'only: :index', so when we  add more action (very soon),
we don't forget to add it.